src/mn_menu.c

```diff
@@ -517,6 +517,34 @@ void M_ClearEpisodes(void)
     NewDef.prevMenu = &MainDef;
 }
 
+// Find the nearest map with the definition of an episode.
+
+int MN_GetEpisodeMap(int map)
```

Comment:
Is this function still used anywhere?

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/1833/commits/b92fa2e4be44a3f2c83d1637e8e8dd9c89f228f2